src/System.Text.Json/tests/Utf8JsonReaderTests.TryGet.cs

```diff
@@ -12,6 +12,55 @@ namespace System.Text.Json.Tests
 {
     public static partial class Utf8JsonReaderTests
     {
+        // Test string, Argument to DateTime(Offset).Parse(Exact)
+        public static IEnumerable<object[]> ValidISO8601Tests()
```

Comment:
@BrennanConroy, is this sufficient flexibility to remove your DateTime.Parse workaround?
https://github.com/aspnet/AspNetCore/blob/brecon/jsonp/src/SignalR/common/Protocols.Json/src/Protocol/JsonHubProtocol.cs#L688-L691

Reply:
It'll be a nicer workaround for sure. The real ask is to get this into the serializer.